### DISCUSSION TITLE
This invoice can no longer be paid. Consider voiding, marking as uncollectible, or marking as paid out of band instead.

### DISCUSSION THREAD

**Discussion by myperfectfit-dev** [vercel/community#4510] [Account Issue]
### Summary

We are having very hard time in paying the invoice payment to vercel..

New card addition failing
Old card payment failing
Support escalation not workign
Our systems are down
No one from sales team respoinding to upgrade the account to enterprise
Support team in the discusion forum sending us the same email link to pay which is failing agian again again agianagin
2 days we are having issue.

### Steps to Reproduce

Go to Payment page add newcard showing error
This invoice can no longer be paid. Consider voiding, marking as uncollectible, or marking as paid out of band instead.

Pay the invoice with existing card showing error
This invoice can no longer be paid. Consider voiding, marking as uncollectible, or marking as paid out of band instead.

**Comment by myperfectfit-dev** [comment]
I am trying to reach you to resolve our issue of the payment to vercel [#00166188](https://vercel.com/teams/mpf-dev/support/cases/00166188)

I am getting the following error

This invoice can no longer be paid. Consider voiding, marking as uncollectible, or marking as paid out of band instead.



The email which you sent to accounts team is also showing error.





Can you please send me a link where i can enter own card details and pay



Thanks,

Rajesh KUmar.

**Comment by amyegan** [comment]
More information was provided on your support ticket. Please continue to follow up with that ticket. We cannot safely discuss payment information here in public.

Closing as a duplicate of https://github.com/vercel/community/discussions/4507
